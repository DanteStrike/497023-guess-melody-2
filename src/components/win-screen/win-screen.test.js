import React from "react";
import renderer from "react-test-renderer";
import WinScreen from "./win-screen";

it(`render correctly WinScreen component`, () => {
  const component = renderer
    .create(
        <WinScreen
          gameDuration={0}
          mistakes={0}
          result={0}
          fastAnswersAmount={0}
          onResetGameClick={jest.fn()}
        />
    )
    .toJSON();

  expect(component).toMatchSnapshot();
});
