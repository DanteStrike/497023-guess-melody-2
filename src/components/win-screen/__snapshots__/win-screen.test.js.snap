// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly WinScreen component 1`] = `
<section
  className="login"
>
  <div
    className="login__logo"
  >
    <img
      alt="Угадай мелодию"
      height="83"
      src="img/melody-logo.png"
      width="186"
    />
  </div>
  <h2
    className="login__title"
  >
    Вы настоящий меломан!
  </h2>
  <p
    className="login__total"
  >
    За 
    00
     минуты и 
    00
     секунд вы набрали 
    0
     баллов (
    0
     быстрых), совершив 
    0
     ошибки
  </p>
  <p
    className="login__text"
  >
    Хотите сравнить свой результат с предыдущими попытками? Представтесь!
  </p>
  <form
    action=""
    className="login__form"
  >
    <p
      className="login__field"
    >
      <label
        className="login__label"
        htmlFor="name"
      >
        Логин
      </label>
      <input
        className="login__input"
        id="name"
        name="name"
        type="text"
      />
    </p>
    <p
      className="login__field"
    >
      <label
        className="login__label"
        htmlFor="password"
      >
        Пароль
      </label>
      <input
        className="login__input"
        id="password"
        name="password"
        type="text"
      />
      <span
        className="login__error"
      >
        Неверный пароль
      </span>
    </p>
    <button
      className="login__button button"
      type="submit"
    >
      Войти
    </button>
  </form>
  <button
    className="replay"
    onClick={[MockFunction]}
    type="button"
  >
    Сыграть ещё раз
  </button>
</section>
`;
